Compiler.cmake

```diff
@@ -11,6 +11,7 @@
 # Input Variables
 # * COMMON_MINIMUM_GCC_VERSION check for a minimum gcc version, default 4.4
 # * COMMON_USE_CXX03 When set, do not enable C++11 language features
+# * COMMON_DISABLE When set, do not set any flags
```

Comment:
Both Juan and I have to do all hands stuff today, so time is tight.  Maybe another day this week?